src/checkstyle/checkstyle.xml

```diff
@@ -148,13 +148,6 @@
 					  value="Line has leading space characters; indentation should be performed with tabs only."/>
 			<property name="ignoreComments" value="true"/>
 		</module>
-		<module name="RegexpSinglelineJava">
-			<property name="maximum" value="0"/>
-			<property name="format" value="org\.mockito\..*Mockito\.(when|doThrow|doAnswer)"/>
-			<property name="message"
-					  value="Please use BDDMockito instead of Mockito.(when|doThrow|doAnswer)."/>
```

Comment:
If you don't mind I would like to remove the forced use of `BDDMockito`. This is something in most Spring projects that I do not understand the need for. Mockito (vanilla) has always been enough for me to do what I need and then some. Thoughts?

Reply:
Ya, I agree with that assessment. I just followed what other projects are doing. But Mockito is sufficient enough for our testing.